app/assets/stylesheets/darkswarm/shop_tabs.scss

```diff
@@ -9,7 +9,7 @@
     color: $dark-grey;
     box-shadow: $distributor-header-shadow;
     position: relative;
-    z-index: 10;
+    z-index: 21; // +1 from .darkswarm navigation
```

Comment:
I guess that we could use variables to make the relationship clear.

Reply:
Done.